_collections/_section_ios/swift_ui.md

```diff
@@ -0,0 +1,100 @@
+---
+title: SwiftUI
+position: 9
+type: iOS
+description:
+right_code: >-
+  {: title="SwiftUI" }
+---
+
+Integrating the SDK into a SwiftUI project is simple:
```

Comment:
My suggestion would be rephrase from "is simple" to 
`Integrating the SDK into a SwiftUI project can be done in the following steps:`

This is better phrasing as you've outlined below there are steps to follow. I'd leave to you guys to see what you think.